pkg/controller/status/controller.go

```diff
@@ -440,13 +461,15 @@ func (c *Controller) updateControllerConditionByReason(cs *conditions,
 func (c *Controller) checkDisabledGathering() {
 	// disabled state only when it's disabled by config. It means that gathering will not happen
 	if !c.configurator.Config().DataReporting.Enabled {
-		c.ctrlStatus.setStatus(DisabledStatus, noTokenReason, reportingDisabledMsg)
+		c.ctrlStatus.setStatus(DisabledStatus, noTokenReason, reportingDisabledMessage)
+		c.ctrlStatus.setStatus(DisabledInAPIStatus, noTokenReason, "gathering is disabled by removing the token")
```

Comment:
> "gathering is disabled by removing the token"

I think we can be more specific here as well.

Reply:
I have reused here this message `Gathering disabled by removing the pull-secret`

Reply:
So `Gathering is disabled by removing the cloud.openshift.com field from the pull secret` should be enough to explain this condition?

Reply:
Sounds good to me.